samples/githubDispatch/command.js

```diff
@@ -0,0 +1,51 @@
+const { Octokit } = require('octokit')
+// argument format
+// /fusebot <user/repo> <workflow_id> <branch/tag name>
```

Comment:
This is already in readme and help - remove?

Reply:
Done